parachain/runtimes/gargantua/src/lib.rs

```diff
@@ -572,6 +571,30 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+impl pallet_hyperbridge::Config for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type IsmpHost = Ismp;
+}
```

Comment:
Hyperbridge runtimes don't need this

Reply:
it needs it as token-gatway config is tightly coupled with it